### ISSUE TITLE
papercut: unify title of icon-more

### ISSUE THREAD

**Issue by cloud2018** [nextcloud/deck#688]
icon-more has title / aria-label "Actions", "More action" or "View more" or not labeled.
part.board.mainView.php, part.card.attachments.php and part.navigation.php

What do you think about always  use title "More actions"?

**Comment by juliushaertl** [comment]
I think just "Actions" is the best option. We should probably use more actions if there were another action button besides the three dots menu, but we don't have that right now.
